clients/src/main/java/org/apache/kafka/common/requests/BrokerRegistrationRequest.java

```diff
@@ -57,6 +57,10 @@ public BrokerRegistrationRequestData data() {
         return data;
     }
 
+    public int brokerId() {
+        return data.brokerId();
+    }
+
```

Comment:
I think this pattern undermines the advantages of the message JSON schema and the Java code generated from it.